src/PostgREST/Error.hs

```diff
@@ -289,6 +291,9 @@ instance JSON.ToJSON Error where
   toJSON (JwtTokenInvalid message) = JSON.object [
     "message" .= (message :: Text)]
   toJSON NotFound = JSON.object []
+  toJSON RpcNotFound = JSON.object [
+    "hint"    .= ("No function matches the given name and argument types. If a new function was created in the database with this name, try updating the schema cache." :: Text),
```

Comment:
```suggestion
    "hint"    .= ("No function matches the given name and arguments. If a new function was created in the database with this name, try reloading the schema cache." :: Text),
```

Reply:
I'll use this and @steve-chavez 's feedback to change the error message.